pkg/testing/integration/program.go

```diff
@@ -1152,6 +1194,37 @@ func (pt *programTester) prepareNodeJSProject(projinfo *engine.Projinfo) error {
 
 }
 
+// readPackageJSON unmarshals the package.json file located in pathToPackage. It ensures the returned object
+// has "dependencies", "devDependencies", and "peerDependencies" sections.
+func readPackageJSON(pathToPackage string) (map[string]interface{}, error) {
```

Comment:
The second sentence in the  comment doesn't appear to be correct (anymore?).

Reply:
Yes, fixed.